spec/unit/api_spec.rb

```diff
@@ -114,6 +114,30 @@
     end
   end
 
+  describe ".parse_environmentdir" do
+
+    it "calls parse_deployed_env on each dir in the path" do
+      @env_path = '/tmp/environments'
```

Comment:
Actually, it looks like there's not a lot of reuse in variables between tests; if a variable is only used in a single test it should be a local variable instead of an instance variable so that it can't accidentally leak between tests.